apps/costcontrol/js/message_handler.js

```diff
@@ -133,8 +133,8 @@
   }
   window.getTopUpTimeout = getTopUpTimeout;
 
-  function addNetworkUsageAlarm(dataInterface, dataLimit, callback) {
-    NetworkUsageAlarm.updateAlarm(dataInterface, dataLimit, callback);
+  function addNetworkUsageAlarm(dataInterface, dataLimit, onsuccess, onerror) {
+    NetworkUsageAlarm.updateAlarm(dataInterface, dataLimit, onsuccess, onerror);
```

Comment:
I'm not sure if passing `null` as callback may cause problems. Double check it and if it does, pass an empty function instead. After using promises I started to hate callbacks, ;)